addons/binding/org.openhab.binding.yamahareceiver/ESH-INF/thing/thing-types.xml

```diff
@@ -1,121 +1,384 @@
-                <option value="PLII[x] Game">PLII[x] Game</option>
-                <option value="PLII[x] Music">PLII[x] Music</option>
-                <option value="PLII[x] Movie">PLII[x] Movie</option>
-                <option value="7ch Enhancer">7ch Enhancer</option>
-
-                <option value="2ch Stereo">2ch Stereo</option>
-                <option value="5ch Stereo">5ch Stereo</option>
-                <option value="7ch Stereo">7ch Stereo</option>
-                <option value="STRAIGHT">STRAIGHT</option>
-                <option value="Straight Enhancer">Straight Enhancer</option>
-            </options>
-        </state>
-    </channel-type>
-    
-    <channel-type id="volume">
-        <item-type>Dimmer</item-type>
-        <label>Volume</label>
-        <description>Set the volume level</description>
-        <category>SoundVolume</category>
-    </channel-type>
-    
-    <channel-type id="volumeDB" advanced="true">
-        <item-type>Number</item-type>
-        <label>Volume in dB</label>
-        <description>Set the volume level (dB)</description>
-        <category>SoundVolume</category>
-        <state min="-80" max="12" step="0.5" pattern="%.1f dB" />
-    </channel-type>
-    
-    <channel-type id="mute">
-        <item-type>Switch</item-type>
-        <label>Mute</label>
-        <description>Enable/Disable Mute on the AVR</description>
-    </channel-type>
-    
+<thing:thing-descriptions bindingId="yamahareceiver" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="yamahaAV">
+		<label>Yamaha Receiver</label>
+		<description>Yamaha Receiver of product line CX-A5000, RX-A30xx, RX-A20xx, RX-A10xx, RX-Vxxx, RX-Z7, DSP-Z7, RX-S600, HTR-xxxx</description>
+
+		<channels>
+			<channel id="power" typeId="power" />
+		</channels>
+
+		<properties>
+			<property name="version" />
+			<property name="assigned_name" />
+		</properties>
+
+		<config-description>
+			<parameter name="HOST" type="text" required="true">
+				<context>network-address</context>
+				<label>Address</label>
+				<description>The address of the AVR to control.</description>
+			</parameter>
+			<parameter name="REFRESH_IN_SEC" type="integer" required="false">
+				<label>Refresh interval</label>
+				<description>Refresh interval in seconds.</description>
+				<default>60</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="zone">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="yamahaAV" />
+		</supported-bridge-type-refs>
+		<label>Yamaha Receiver Zone</label>
+		<description>Yamaha Receiver Zone</description>
+
+		<channel-groups>
+			<channel-group typeId="zone_channels" id="zone_channels" />
```

Comment:
@davidgraeff I didn't notice it when I reviewed that PR, I only came across this right now: Why did you introduce channel-groups here? Those are meant for reusing the same set of channels multiple times, but you already chose to do this by Things. So I do not see any good reason to have channel groups here. Would it be possible to get rid off them?

Reply:
The channel group usage is a leftover of a previous design, it works without them yes. A change touches some areas in the code and renders current configs invalid, I guess. I need to find out who is working on this add-on at the moment, there were some proposed changes in the community forum.